README.md

```diff
@@ -27,8 +27,8 @@ Tools built on DynamoRIO include:
   [drcpusim](http://dynamorio.org/docs/page_drcpusim.html)
 - The "strace for Windows" tool [drstrace](http://drmemory.org/strace_for_windows.html)
 - The code coverage tool [drcov](http://dynamorio.org/docs/page_drcov.html)
-- The library tracing tool [drltrace](http://drmemory.org/docs/page_drltrace.html)
```

Comment:
@mxmssh -- does that work for you?  It is a step down from the html page but you could spice up that drltrace/README.md page separately.